content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -77,7 +77,7 @@ For a reference to old feature gates that are removed, please refer to
 | `CSIMigrationPortworx` | `false` | Alpha | 1.23 | 1.24 |
 | `CSIMigrationPortworx` | `false` | Beta | 1.25 | |
 | `CSIMigrationRBD` | `false` | Alpha | 1.23 | |
-| `CSINodeExpandSecret` | `false` | Alpha | 1.25 | |
+| `CSINodeExpandSecret` | `true` | Beta | 1.27 | |
```

Comment:
The `Alpha` stage line should be kept instead of being removed. We still need to let users know that there was an Alpha stage that spans from 1.25 to **1.26**. Check how we dealt with other features for reference.